src/lib/objects.js

```diff
@@ -0,0 +1,87 @@
+const createPerson = (name, age) => {
+  const person = {};
+  person.name = name;
+  person.age = age;
+  return person;
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  if (property === 'age') {
+    return object.age;
+  }
+  return object.name;
```

Comment:
This is not a good way of doing this. It passes the tests now, but what would happen if you added more properties to the object or changed one of their names? It would no longer work. Instead, you should abstract this down another layer:
```
  return object[property];
```